server/src/metadata/object-metadata/dtos/object-metadata.dto.ts

```diff
@@ -47,13 +47,13 @@ export class ObjectMetadataDTO {
   @Field({ nullable: true })
   icon: string;
 
-  @Field()
+  @FilterableField()
```

Comment:
I realized that the filtering was being done on the frontend, which could create issues with pagination. I'm introducing these two filters, along with the 'isSystem' filter, for future improvements.